plugins/scaffolder-actions/scaffolder-backend-module-utils/src/types.ts

```diff
@@ -23,59 +23,89 @@ export const yamlOptionsSchema = {
   description: 'YAML stringify options',
   type: 'object',
   properties: {
-    indent: {
-      description: '(default: 2) - indentation width to use (in spaces)',
-      type: 'number',
+    blockQuote: {
```

Comment:
`commentString` was omitted in the [original PR](https://github.com/RoadieHQ/roadie-backstage-plugins/pull/1241/commits/42db6b8f18c5e1b1ba3e685545b63118ed54ef4f#diff-0c7f5cb766fd6b51674c33ddddd04f5d7cc7aa0a906a690c4f9854d34c1eca19R19) - I'm not sure the reasoning but I've respected that decision by omitting it from the `yamlOptionsSchema` and updated test inputs.